engine/classes/Elgg/Database/UsersTable.php

```diff
@@ -240,23 +198,11 @@ public function removeAdmin($user_guid) {
 			return false;
 		}
 
-		$query = "
-			UPDATE {$this->table}
-			SET admin = 'no'
-			WHERE guid = :guid
-		";
-
-		$params = [
-			':guid' => (int) $user_guid,
-		];
+		$user->admin = 'no';
```

Comment:
Thinking out loud: name collisions are not really possible, because you wouldn't be able to write metadata with that name, unless you used `create_metadata()`?